plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java

```diff
@@ -212,4 +248,63 @@ public MavenPreferencePage setHideFoldersOfPhysicalyNestedModules(boolean check)
 		new CheckBox(this, MavenPreferencePage.HIDE_FOLDERS_OF_PHYSICALLY_NESTED_MODULES).toggle(check);
 		return this;
 	}
+	
+	/**
+	 * Returns true when Use Maven-Central to identify unknown artifacts checkbox is checked .
+	 *
+	 * @return true, if is Use Maven-Central to identify unknown artifacts checked
+	 */
+	public boolean isUseMavenCentralToIdentifyUnkArtifChecked() {
+		return new CheckBox(this, MavenPreferencePage.USE_MAVEN_CENTRAL_TO_IDENTIFY_UNKNOWN_ARTIFACTS).isChecked();
+	}
+	
+	/**
+	 * Sets Use Maven-Central to identify unknown artifacts checkbox .
+	 *
+	 * @param check the new Use Maven-Central to identify unknown artifacts
+	 */
+	public MavenPreferencePage setUseMavenCentralToIdentifyUnkArtif(boolean check) {
```

Comment:
Too long naming. I would make it shorten and use javadoc for wider description.

Reply:
outdated